src/PaymentMethod/PayonePaypalV2.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PayonePayment\PaymentMethod;
+
+use PayonePayment\Installer\PaymentMethodInstaller;
+use PayonePayment\PaymentHandler\PayonePaypalV2PaymentHandler;
+
+class PayonePaypalV2 extends AbstractPaymentMethod
+{
+    final public const UUID = PaymentMethodInstaller::PAYMENT_METHOD_IDS[self::class];
+
+    final public const TECHNICAL_NAME = 'payone_paypal_v2';
+
+    protected string $id = self::UUID;
+
+    protected string $name = 'PAYONE PayPal v2';
+
+    protected string $description = 'Pay easily and secure with PayPal v2.';
```

Comment:
I have removed it from `name` and `description` and added the `PaymentDistinguishableNameEventListener`, which inserts the “v2” into the `distinguishableName`, which is only displayed in the administration. This means that no “v2” is displayed in the storefront, but it is in the administration, so that the payment methods can be distinguished in the administration.